### ISSUE TITLE
Ability to change the logging level for the network policy agent container

### ISSUE THREAD

**Issue by uditsidana** [aws/aws-network-policy-agent#270]
**What would you like to be added**: Ability to control the logging level for network policy agent container.

**Why is this needed**:
As the default logging level for network policy agent container is set to "info", for long running nodes this can accumulate a lot of logs. Therefore, could we pass in an env variable just like ipamd or an argument to tune down the logging level.

**Comment by atilsensalduz** [comment]
That's what the **log-level** argument is for, isn't it?

https://github.com/aws/aws-network-policy-agent/blob/dc9a6a82adf6b3d83956830c6e4451fb1e5da820/pkg/config/controller_config.go#L6

**Comment by haofeif** [comment]
Agreed.  Can we allow the users to set the log level accordingly? And by default maybe not requiring to INFO, as there are way too many logs for it. With or without forwarding to cloud watch, it will be quite unnecessary. 

Also, will the logs be rotating on the worker nodes ?

**Comment by Pavani-Panakanti** [comment]
We have PR merged to improve logging in NP agent and also configure log levels https://github.com/aws/aws-network-policy-agent/pull/409. Readme has details on how cx can configure the logging level. This will be available in next release v1.2.2
